azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTest2.java

```diff
@@ -118,6 +123,18 @@ public void testBasicRun() throws Exception {
     assertEquals("test2.6", joba.get("param6"));
     assertEquals("test2.7", joba.get("param7"));
     assertEquals("test2.8", joba.get("param8"));
+    assertThat(joba.get(CommonJobProperties.JOB_ID)).isEqualTo("joba");
```

Comment:
Just curious, any reason to not use plain `assertEquals`?

Reply:
We're trying to use Assertj for newer tests: https://github.com/azkaban/azkaban/blob/master/CONTRIBUTING.md.